docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,113 @@
+# Production checklist for ScalarDB Cluster
+
+This checklist provides recommendations when deploying ScalarDB Cluster in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDB Cluster on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up ScalarDB Cluster in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes in three availability zones and deploy at least three pods.
```

Comment:
I feel `three avaliability zones` might be too strict.
I think it highly depends on the applications' requirements, so how about something like this?
```suggestion
To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes and deploy at least three pods spread across the worker nodes. It depends on the application's requirements, but we also recommend users to place the worker nodes in different availability zones (AZs) to survive an AZ failure.
```